src/interaction-design/typography.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Typography
+permalink: /:path/:basename/
+---
+There are several typography elements that can enhance the accessibility of web content. These elements include:
+- Font size, line height, line length
+- Font weight
+- Typeface
+
+You may wonder why we discuss these three elements together, while others are mentioned separately. This is because font size, line height, and line length are interrelated and adjusting one will require reevaluating the balance with the others. They cannot be isolated when considering accessibility. And this time we will look at body text first, and then heading text after.
+
+## Body text
+
+### Font Size
+There have been various studies on the ideal font size for web content, and it is suggested to use 18-22 px for desktop and 16-18 px for mobile. This range is large enough to be readable in most situations. The GOV.UK Design System, a widely used front-end framework for public sector services, uses 19px for large screens, which is within this range. The final font size may also depend on the typeface chosen, but this recommendation should be suitable for most common web fonts.
```

Comment:
A few instances of headings with space before the paragraph - would be good to include a space between them, but we can fix this later if it's not obvious how to do it in the CMS